vpr/src/base/CheckSetup.h

```diff
@@ -13,6 +10,6 @@ void CheckSetup(const t_packer_opts& PackerOpts,
                 const t_det_routing_arch& RoutingArch,
                 const std::vector<t_segment_inf>& Segments,
                 const t_timing_inf& Timing,
-                const t_chan_width_dist Chans);
+                const t_chan_width_dist& Chans);
```

Comment:
If you have the energy for it, these variables that start with a capital are not in line with typical vtr style -- better to start with a lower case variable. But they're not your code so if you want to leave them it's OK.